ansible/playbooks/tools/upgrade-reboot.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+# upgrade - upgrade packages which are safe to be updated.
+# This means no packages will be removed.
+# The machine is rebooted if necessary. Skip the reboot
+# tag to avoid that.
+# Copyright (C) 2020 Ellen Papsch <ellenpapsch@gmail.com>
+# SPDX-License-Identifier: GPL-3.0-only
+
+- hosts: debops_all_hosts
+  become: true
+  tasks:
+
+    - name: Upgrade safe packages with refreshed cache
+      apt:
+        update_cache: yes
+        upgrade: safe
+
+    - name: Check if reboot is required
+      stat:
+        path: '/var/run/reboot-required'
+        get_checksum: no
+      register: reboot_required
+
+    - name: Reboot a machine with shutdown command in unusual place
+      reboot:
+        search_paths:
+          - '/lib/molly-guard'
```

Comment:
You should specify the other paths used by default, `['/sbin', '/usr/sbin', '/usr/local/sbin']`. This will alow the playbook to be used on hosts without `molly-guard` installed.